torch/csrc/api/include/torch/data/dataloader.h

```diff
@@ -37,29 +37,20 @@ class DataLoader {
       : options_(std::move(options)),
```

Comment:
Maybe I missed it, but is there a way to get the sampler from a data loader? One thing people often want to do, and we don't really support properly in Python API, is to dynamically update the sampler.

Reply:
I don't think that would work too well because `Sampler` is a template parameter, so it's type is rather hard to change any way

Reply:
Oh by dynamic I mean to change the state of the sampler. For instance, adjust an internal state vector of the sampler.

Reply:
I can provide access to it yes